examples/pennylane_run_quantum_natural_gradient.py

```diff
@@ -0,0 +1,356 @@
+
+The problem with the above approach is that each optimization step
+is highly dependent on the euclidean geometry of the parameter space.
+The parametrization chosen is not necessarily unique; the scale of the
+parameters, or even the parametrization used, could easily be modified.
+If we instead consider the optimization problem as a distribution of possible
+output values given an input, a better approach is to instead perform the
+gradient descent in the *distribution space*, which is invariant with respect
+to the parametrization.
+
+In classical neural networks, the above process is known as
+*natural gradient descent*, and was first introduced by
+`Shun-Ichi Amari (1998) <https://www.mitpressjournals.org/doi/abs/10.1162/089976698300017746>`__.
+The standard gradient descent is modified as follows:
+
+.. math:: \theta_{t+1} = \theta_t - \eta F^{-1}\nabla \mathcal{L}(\theta),
+
+where :math:`F` is the `Fisher information matrix <https://en.wikipedia.org/wiki/Fisher_information#Matrix_form>`__.
+The Fisher information matrix acts as a metric tensor, transforming the
+steepest descent in the euclidean parameter space to the steepest descent in the
+distribution space.
+
+The quantum analogue
+^^^^^^^^^^^^^^^^^^^^
+
+In a similar vein, it has been shown that the standard euclidean geometry
+is sub-optimal for optimization of quantum variational algorithms.
+The space of quantum states instead possesses a unique invariant metric
+tensor known as the Fubini-Study metric tensor :math:`g_{ij}`, which can be used to
+construct a quantum analogue to natural gradient descent:
+
+.. math:: \theta_{t+1} = \theta_t - \eta g^{+}(\theta_t)\nabla \mathcal{L}(\theta),
+
+where :math:`g^{+}` refers to the pseudo-inverse.
+
+.. note::
+
+    It can be shown that the Fubini-Study metric tensor reduces
+    to the Fisher information matrix in the classical limit.
+
+    Furthermore, in the limit where :math:`\eta\rightarrow 0`,
+    the dynamics of the system are equivalent to imaginary-time
+    evolution within the variational subspace, as proposed in
+    `McArdle et al. (2018) <https://arxiv.org/abs/1804.03023>`__.
+"""
+
+##############################################################################
+# Block-diagonal metric tensor
+# ----------------------------
+#
+# The block-diagonal approximation to the Fubini-Study metric tensor
+# of a variational quantum circuit can be evaluated on quantum hardware.
+#
+# Consider a quantum node represented by the variational quantum circuit
+#
+# .. math::
+#
+#     U(\mathbf{\theta}) = W(\theta_{i+1}, \dots, \theta_{N})X(\theta_{i})
+#     V(\theta_1, \dots, \theta_{i-1}),
+#
+# where all parametrized gates can be written of the form :math:`X(\theta_{i}) = e^{i\theta_i K_i}`.
+# That is, the gate :math:`K_i` is the *generator* of the parametrized operation :math:`X(\theta_i)`
+# corresponding to the :math:`i`-th parameter.
+#
+# For each parametric layer :math:`\ell` in the variational quantum circuit

```

Comment:
This is where I get confused. Is \ell contained in V, or W, or is it a repetition of U, or is a layer the application of X?

Reply:
Agree that there is a confusing jump here. "Parameteric layer" appears too suddenly without introduction

Reply:
Rewritten this section to properly introduce parametric layers